docs/source/configuration/configuration_basics.md

```diff
@@ -22,7 +22,7 @@ The configuration source folder is [`conf`](../get_started/kedro_concepts.md#con
 ## Configuration environments
 A configuration environment is a way of organising your configuration settings for different stages of your data pipeline. For example, you might have different settings for development, testing, and production environments.
 
-By default, Kedro has a `base` and a `local` environment.
+By default, Kedro Framework has a `base` and a `local` environment.
```

Comment:
I didn't change the wordings too much. In most of the configuration docs, we still saying `OmegaConfigLoader` will load config in `base` and `local`, technically this is only true for Kedro Framework users. I think this will be the majority.

Instead, I add a bit more explanation in the `advance_configuration.md` to explains the different defaults of environment. Feel free to comment on this.

Reply:
I'd maybe go with the following:
```suggestion
By default, Kedro projects have a `base` and a `local` environment.
```